monitor/api/src/main/java/org/datacleaner/monitor/Api.gwt.xml

```diff
@@ -23,6 +23,9 @@
 <module>
 	<inherits name='com.google.gwt.user.User' />
 
+	<source path='dashboard' />
+	<source path='scheduling' />
 	<source path='shared' />
-	
+	<source path='wizard' />
+
 </module>
```

Comment:
CHANGE: This GWT module has to "publish" stuff since it is used from external project.